docs/operations/distributed-tracing.md

```diff
@@ -0,0 +1,60 @@
+# Distributed Tracing
+
+## Modules
+
+Distributed tracing typically requires a mechanism for collecting traces (instrumentation)
+and a separate mechanism for reporting and visualizing those traces.
+
+* [OpenTelemetry](https://opentelemetry.io/) provides instrumentation in multiple languages
+including Node. OpenTelemetry is not yet released as a final stable version and is in Beta.
+OpenTelemetry replaces both OpenTracing and OpenCensus, neither of which are actively being
+developed (security fixes only).
+
+* [Jaeger](https://www.jaegertracing.io/) provides visualization of distributed traces.
```

Comment:
I think we can move the discussion of the back ends to the guidance section as we've generally not made recommendations on that front, only the Node.js components.